### PR TITLE
Added tests for the subjects page

### LINKED ISSUE #102: (SP: 1) Write unit test for "Subjects" page
### Page unit test
Unit test for "**Subjects**" page
Scenaries descriptions:
- [ ] It should render page
- [ ] Text of the page must be in the document

[Link to component](https://github.com/Space2Study-UA-1125/frontEnd/blob/develop/src/pages/subjects/Subjects.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by holus-bolus**
Added unit tests for the subjects page

image

### CODE REVIEW

File: tests/unit/pages/subjects/Subjects.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+vi.mock('~/components/page-wrapper/PageWrapper', () => {
+  const PageWrapperMock = ({ children }) => <div>{children}</div>
+  return {
+    __esModule: true,
+    default: vi.fn().mockImplementation(PageWrapperMock)
+  }
+})
+
+describe('Subjects Component', () => {
+  test('renders "Subjects" text within the PageWrapper', () => {
```
Comment (Wiret02) [inline tests/unit/pages/subjects/Subjects.spec.jsx]: Use word it, not test in test('renders "Subjects" text within the PageWrapper', () => {

**Review by Wiret02 — CHANGES_REQUESTED** [review]

File: tests/unit/pages/subjects/Subjects.spec.jsx (hunk 1 above)
Comment (holus-bolus) [reply tests/unit/pages/subjects/Subjects.spec.jsx]: Done, thanks
image

**Review by Wiret02 — APPROVED** [review]

**Review by VictoriaRodik — APPROVED** [review]

**Review by OliaHrytsak — APPROVED** [review]

**Review by ashfox21 — APPROVED** [review]

File: tests/unit/pages/subjects/Subjects.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import Subjects from '~/pages/subjects/Subjects'
+import { ModalProvider } from '~/context/modal-context'
```
Comment (vlad-khrychov) [inline tests/unit/pages/subjects/Subjects.spec.jsx:4]: I do not like using a real provider here, but can agree with it for speeding up our developing process.

File: tests/unit/pages/subjects/Subjects.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  }),
+  ModalProvider: vi.fn(({ children }) => <div>{children}</div>)
+}))
+
+vi.mock('~/components/page-wrapper/PageWrapper', () => {
+  const PageWrapperMock = ({ children }) => <div>{children}</div>
+  return {
+    __esModule: true,
+    default: vi.fn().mockImplementation(PageWrapperMock)
+  }
```
Comment (vlad-khrychov) [inline tests/unit/pages/subjects/Subjects.spec.jsx:19]: It should be work easier 
```suggestion
  return {
    __esModule: true,
    default:  ({ children }) => <div>{children}</div>
  }
```

**Review by vlad-khrychov — APPROVED** [review]
Let's merge it to improve code coverage

**Pull request merged**